plugin/trino-phoenix5/src/main/java/io/trino/plugin/phoenix5/PhoenixMetadata.java

```diff
@@ -86,6 +91,22 @@ public JdbcTableHandle getTableHandle(ConnectorSession session, SchemaTableName
                 .orElse(null);
     }
 
+    @Override
+    public ConnectorTableProperties getTableProperties(ConnectorSession session, ConnectorTableHandle table)
+    {
+        List<LocalProperty<ColumnHandle>> sortingProperties = ImmutableList.of();
+        JdbcTableHandle tableHandle = (JdbcTableHandle) table;
+        if (!tableHandle.getSortOrder().isEmpty()) {
+            sortingProperties = tableHandle.getSortOrder().get()
+                    .stream()
+                    .map(item -> new SortingProperty<ColumnHandle>(
+                            item.getColumn(),
+                            item.getSortOrder()))
+                    .collect(toImmutableList());
+        }
+        return new ConnectorTableProperties(TupleDomain.all(), Optional.empty(), Optional.empty(), Optional.empty(), sortingProperties);
```

Comment:
Please add a test which uses this functionality in TestPhoenixConnectorTest

Reply:
You mean a test that verifies the query plan? (The results will be identical)

Reply:
I mean a test along the lines of TestHiveConnectorTest#testUseSortedProperties

Reply:
Good. I had started basically replicating that test.